spring-xd-rest-client/src/main/java/org/springframework/xd/rest/client/ResourceOperations.java

```diff
@@ -50,4 +50,10 @@
 	 */
 	public void destroyAll();
 
+	/**
+	 * Delete Queues and DLQs for a stream, and any exchanges created for taps.
```

Comment:
Interesting - maybe we'll find a way to leverage that for Kafka topics too. When, you know, deletion works or something.